### DISCUSSION TITLE
New to speaker diarization

### DISCUSSION THREAD

**Discussion by temitopefunmi** [speechbrain/speechbrain#1821] [Q&A]
I want to implement the example of speaker diarization on my mac.
I have python 3.8 installed in a new environment on Anaconda.
I have installed pyannote audio and import pretrained speaker embedding.
The embedding model is PretrainedSpeakerEmbedding("speechbrain/spkrec-ecapa-voxceleb").
But I get the error that speechbrain has to be installed. I have checked the installed packages on this environment and I saw that speechbrain is involved.
Please help with this.
Thanks in advance.

**Comment by anautsch** [comment]
Hi @temitopefunmi hm; let's see what can be done. What do you get after this one?
```
pip list | grep speech
```
Please ensure your conda environment w/ SpeechBrain is active.

Do you get a `ModuleNotFoundError` error when running this one?
```
python -c 'import speechbrain'
```

**Reply by temitopefunmi** [reply]
Thank you so much for the quick response. I apologize for delay in responding.
Here is what i got.
![Untitled](https://user-images.githubusercontent.com/53179810/215705820-af17be3a-3495-420f-82ee-c109bbdca9da.png)

**Comment by anautsch** [comment]
This looks like #1603 (`Symbol not found: ____chkstk_darwin` for sentencepiece).

@AsuMagic provided a good suggestion there, please take a look:
https://github.com/speechbrain/speechbrain/issues/1603#issuecomment-1279742128

**Reply by temitopefunmi** [reply]
Thank you
